src/engines/nwn2/roster.h

```diff
@@ -40,13 +40,19 @@ class Roster {
 	Roster();
 	~Roster();
 
+	/** Cycle through the roster members */
```

Comment:
The Doxygen comment gets applied to only the next method, not the whole group. So this comment would be a bit confusing.

You could move it to `getNextRosterMember()`, that would make more sense. (Also, please end Doxygen comments with a period.)